hera_mc/cm_partconnect.py

```diff
@@ -986,7 +986,7 @@ def update_connection(session=None, data=None, add_new_connection=False):
                 connection = None
             else:
                 connection = conn_rec.first()
-        else:
+        else:  # pragma: no cover
             print("Error:  more than one of ", dkey, " exists (which should not happen).")
```

Comment:
I agree. This is @david-deboer's part of the code, so I'll let him respond.

Reply:
It would be an error.  It shouldn't be able to happen, given unique primary keys, but I left it in just in case reality is more clever than me in making it happen.  That is why the pragma -- I couldn't come up with a way to test it.  Bryna, since you are editing, can you raise a runtime error?  The other option is to remove since it shouldn't be able to happen, but then it would be silent.